Snakefile

```diff
@@ -442,7 +456,12 @@ Runs Trinity script to get statistics about the assembled transcriptome
     1
   shell:
     """
-    {TRINITY_HOME}/util/TrinityStats.pl {input.transcriptome} > {output.stats} 2> {log}
+    assembler=$( grep '^assembler: ' config.yaml | sed -e 's/^assembler: "\(.*\)"/\1/' )
```

Comment:
No need to parse the config file manually here. See
https://stackoverflow.com/questions/49140578/how-can-one-access-snakemake-config-variables-inside-shell-section